roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -49,3 +49,9 @@ edpm_bootstrap_network_service: NetworkManager
 
 # String for SELinux state. One of: disabled, enforcing, permissive
 edpm_bootstrap_selinux_mode: enforcing
+
+# Swap management
+edpm_bootstrap_swap_size_megabytes: 4096
+edpm_bootstrap_swap_path: /swap
+edpm_bootstrap_swap_partition: false
+edpm_bootstrap_swap_partition_label: swap1
```

Comment:
I think we should hard-fail if the partition with the given label is not found.

Reply:
There is a fallback path https://github.com/bogdando/edpm-ansible/blob/f6144dd7cc0d767d2469a8922d93042a92bbaeaf/roles/edpm_bootstrap/tasks/swap.yml#L50 in such a case.
Please rememver, this is a direct port of the tripleo supported feature. We can adjust it in followups